sdk/core/azure-core/src/http/logging_policy.cpp

```diff
@@ -30,22 +30,69 @@ std::string TruncateIfLengthy(std::string const& s)
   return s.substr(0, BeginLength) + Ellipsis + s.substr(length - EndLength, EndLength);
 }
 
-std::string GetRequestLogMessage(Request const& request)
+std::string GetRequestLogMessage(
+    Azure::Core::Http::LoggingPolicyOptions const& options,
+    Request const& request)
 {
   std::ostringstream log;
 
-  log << "HTTP Request : " << HttpMethodToString(request.GetMethod()) << " "
-      << request.GetUrl().GetAbsoluteUrl();
+  std::string url = request.GetUrl().GetAbsoluteUrl();
```

Comment:
Since you are calling GetUrl() more than once in this function, consider doing
auto const& url = request.GetUrl();  here and then using it below